lib/vsc/utils/fancylogger.py

```diff
@@ -101,6 +174,13 @@
 
 DEFAULT_UDP_PORT = 5005
 
+# poor man's enum
```

Comment:
???? 

How can you simulate a "enum" (algebraic "sum" type) with a namedtuple (algebraic "product" type)?